saml-core/src/main/java/org/keycloak/saml/processing/core/util/XMLSignatureUtil.java

```diff
@@ -41,7 +41,7 @@
 import org.xml.sax.SAXException;
 
 import javax.xml.bind.JAXBException;
-import javax.xml.crypto.MarshalException;
+import javax.xml.crypto.*;
```

Comment:
@douglaspalmer I'd suggest to use explicit imports here, as it was before. Probably this is just some IDE configuration.

Reply:
ok, I'll fix that, (IntelliJ did it).